app/views/home/auth_failure.html.haml

```diff
@@ -0,0 +1,19 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl
+      Report management information to CCS
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %p
+      We were unable to sign you in with the provided credentials. Please try again.
+
+    %p
+      %br
+      = link_to 'Sign in', '/auth/auth0', class: 'govuk-button govuk-button--start', id: 'sign-in', 'aria-label' => "Sign in to the service"
+
+    %p
+      If you are unable to sign in or do not receive a password reset email from the service contact report-mi@crowncommercial.gov.uk.
```

Comment:
The email address is inlined here, but also DRY'd up below with the `support_email_address` method. Could we use the method in both places so that changing the support email address only has to happen in one place?

Reply:
I copied this from the guest_homepage view so I'll tidy it up there as well